packages/next/server/node-web-streams-helper.ts

```diff
@@ -111,8 +111,8 @@ export function encodeText(input: string) {
   return new TextEncoder().encode(input)
 }
 
-export function decodeText(input?: Uint8Array) {
-  return new TextDecoder().decode(input)
+export function decodeText(input?: Uint8Array, textDecoder?: TextDecoder) {
+  return textDecoder ? textDecoder.decode(input, { stream: true } ) : new TextDecoder().decode(input)
```

Comment:
Since decodeText is being exported I didn't want to mess with the code signature, so I just added the textDecoder argument.